{{ cookiecutter.project_name }}/pyproject.toml

```diff
@@ -38,13 +38,14 @@ select = ["E4", "E7", "E9", "F", "I", "UP", "C901"]
 [tool.ruff.lint.isort]
 known-first-party = ["{{ cookiecutter.package_name }}"]
 
+
+[tool.ruff.lint.per-file-ignores]
+"__init__.py" = ["F401", "F403"]
```

Comment:
I'd prefer not to have it in the basic template. It is fine for properly build and maintained APIs, but advisors will also use this template and they have the tendency to make an unholy mess with `import *` in init files.

Easy to enable/add for individual projects that need it.

Reply:
Documented it in https://github.com/nens/nens-meta/commit/3a9fa0cfaedefde695492f14b3e5e80e11f82f51